composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":             ">=5.3.2",
         "ext-curl":        "*",
-        "guzzle/guzzle":   "~3.7"
+        "guzzlehttp/guzzle": "~3.7 || ~4.0 || ~5.0 || ~6.0"
```

Comment:
> guzzle 3 does not have the same package name.

Nearly right IMO. Guzzle 3 have a dedicated outdated package.

But, we can retrieve it on the new package: https://packagist.org/packages/guzzlehttp/guzzle#v3.8.1